Automattic/wp-calypso
# [Fraud] Get Fingerprinted Visitor ID

Related to https://github.com/Automattic/gold/issues/461

## Proposed Changes

* Gets FingerprintJS to build
* Adds `X-Fingerprint` header

## Why are these changes being made?


* To support payment processors

## Testing Instructions



* Add an item to checkout (e.g. upgrade your plan or buy a domain)
* View the console to see the visitor ID

## Pre-merge Checklist
